saleor/graphql/order/mutations/order_confirm.py

```diff
@@ -66,10 +66,10 @@ def perform_mutation(cls, root, info, **data):
                 # We use the last transaction as we don't have a possibility to
                 # provide way of handling multiple transaction here
                 payment_transaction = payment_transactions[-1]
-                request_capture_action(
+                request_charge_action(
                     transaction=payment_transaction,
                     manager=info.context.plugins,
-                    capture_value=payment_transaction.captured_value,
+                    charge_value=payment_transaction.authorized_value,
```

Comment:
why `authorized_value` ?

Reply:
This is a logic that calls a charge request action. Previously we were calling capture requests with the amount that is already captured.